dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx
```

Comment:
I want the dev requirements to use the latest available sphinx

Reply:
Generally we stay away from open ended dependencies for external libraries because they could break at any time. I think it's better to specify the latest version and update as new versions of Sphinx are released and we can test with them.